ui/src/main/java/ucar/nc2/ui/GetDataRunnable.java

```diff
@@ -10,7 +10,7 @@
 /**
  *
  */
-interface GetDataRunnable {
+public interface GetDataRunnable {
```

Comment:
This comment applies to the next one also: Is this interface/class being used beyond the enclosing package?  I assume that is why the public modifier is added.

Reply:
Never mind.  I see why.

Reply:
@hvandam2, The `public`is there because NCdumpPanel implements GetDataRunnable.  But NCdumpPanel was moved from being an inner class of ToolsUI to being a separate class in `ucar.nc2.ui.op`, along with all the other classes extending OpPanel.

Reply:
Snap! Looks like we ere typing at the same time.

Reply:
Looks good, I think we should leave the org.springframework classes for a separate change set.  I don't see a need for them, but we should look into it in detail.  I am going to test for a couple of days.

Reply:
Tested on both mac and windows, everything looks great!@mdsoftware Thank you for the excellent work on this.@lesserwhirls Let's get this merged and in, Howard